sdk/keyvault/keyvault-secrets/review/keyvault-secrets.review.api.md

```diff
@@ -0,0 +1,151 @@
+## API Report File for "@azure/keyvault-secrets"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { HttpClient } from '@azure/core-http';
+import { HttpPipelineLogger } from '@azure/core-http';
+import * as msRest from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface DeletedSecret extends Secret {
+    readonly deletedDate?: Date;
+    recoveryId?: string;
+    readonly scheduledPurgeDate?: Date;
+}
+
+// @public
+export type DeletionRecoveryLevel = "Purgeable" | "Recoverable+Purgeable" | "Recoverable" | "Recoverable+ProtectedSubscription";
+
+// @public
+export interface GetSecretOptions {
+    requestOptions?: msRest.RequestOptionsBase;
+    version?: string;
+}
+
+// @public
+export interface GetSecretsOptions {
+    requestOptions?: msRest.RequestOptionsBase;
+}
+
+// @public
+export interface NewPipelineOptions {
+    // (undocumented)
+    HTTPClient?: HttpClient;
+    // (undocumented)
+    logger?: HttpPipelineLogger;
```

Comment:
Can we remove this? With the new logging guidelines this should be unnecessary, logging can be controlled via other means (AZURE_LOG_LEVEL especially). There will be some new options coming to control whitelisting etc.